packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -258,6 +258,21 @@ new UserPool(this, 'myuserpool', {
 
 Note that, `tempPasswordValidity` can be specified only in whole days. Specifying fractional days would throw an error.
 
+#### Advanced Security
+
+User ppols can be configured Advanced security. It can be turned on or used audit mode to gather metrics on detected risks without taking action. In audit mode, the advanced security features publishes metrics to Amazon CloudWatch. 
+See the [documentation on Advanced security](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-advanced-security.html) to
```

Comment:
Looking at the documentation - https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pool-settings-advanced-security.html#cognito-user-pool-configure-advanced-security, once advanced security is set of 'ENFORCED', the user can configure a whole set of additional options.

I can't seem to find these options via CloudFormation. Are you aware of another way to set this up?
How useful is it to provide just the enum without the rest of the options?

Reply:
@nija-at This method (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-cognito-userpoolriskconfigurationattachment.html) is way to set this via CloudFormation.
Is it better to implement it in this PR.

Reply:
I would say so, yes. 
As it is, it's not very useful to the customer to turn on advanced security but not have the option to configure it.

Reply:
I understand what you are saying.
I'm also going to implement an option to set that up.

Reply:
@nija-at I implement an option to setup user pool risk configuration. However, I'm not sure if it should be implemented in `user-pool.ts`, so please review it.